dpdispatcher/ssh_context.py

```diff
@@ -135,7 +135,11 @@ def _setup_ssh(self):
 
         #Begin authentication; note that the username and callback are passed
         if self.totp_secret:
-            ts.auth_interactive(self.username, self.inter_handler)
+            try:
+                ts.auth_interactive(self.username, self.inter_handler)
+            except paramiko.ssh_exception.AuthenticationException:
+                # since the asynchrony of interactive authentication, one addtional try is added
+                ts.auth_interactive(self.username, self.inter_handler)
```

Comment:
Please use [dpdispatcher.utils.retry](https://docs.deepmodeling.com/projects/dpdispatcher/en/latest/api/dpdispatcher.html#dpdispatcher.utils.retry), which can customize the maximum retry times and the sleep time.